src/extensions/default/InlineColorEditor/main.js

```diff
@@ -59,7 +67,45 @@ define(function (require, exports, module) {
         } while (match && (pos.ch < start || pos.ch > end));
 
         if (!match) {
-            return null;
+            // Check if the cursorLine has a CSS rule of type color
+            var cssPropertyName, semiColonPos, colonPos, colorValue, cursorLineSubstring, firstCharacterPos;
+
+            // Get the css property name after removing spaces and ":" so that we can check for it in the file ColorProperties.json
+            cssPropertyName = cursorLine.split(':')[0].trim();
+
+            if (!cssPropertyName || !properties[cssPropertyName]) {
+                return null;
+            }
+
+            if (properties[cssPropertyName].type === "color") {
+                colonPos = cursorLine.indexOf(":");
+                semiColonPos = cursorLine.indexOf(";");
+                if (semiColonPos !== -1) {
+                    // edit the color value of an existing css rule
+                    cursorLineSubstring = cursorLine.substring(colonPos + 1, semiColonPos);
+                    colorValue = cursorLineSubstring.trim();
+                    if (colorRegEx.test(colorValue)) {
+                        firstCharacterPos = cursorLineSubstring.search(/\S/);
+                        pos.ch = colonPos + 1 + firstCharacterPos;
+                        endPos = {line: pos.line, ch: semiColonPos};
+                    } else {
+                        return null;
+                    }
+                } else {
+                    // edit the color value of a new css rule
+                    pos.ch = colonPos + 1;
```

Comment:
Hey.

I understand that a single space looks better visually.
But if the user has more than one space present, he/she must see that many number of spaces after he/she presses Ctrl + E 
Eg : If a user sees this
```js
color:<lots of spaces>red;
```
and places his/her cursor on red and presses Ctrl+E, what he expects is the popup to flow down with the number of spaces retained.
You are limiting the space to one in all cases. 
That is what @flukeout has pointed out in [this](https://github.com/mozilla/brackets/pull/641#issuecomment-290172690) issue.
If the user has a space, it should be retained.